<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prefViewLayout</class>
 <widget class="QWidget" name="prefViewLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View options</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>671</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="title">
    <string>File viewer:</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>651</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Text file suffixes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="textFileSuffixes">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>600</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>681</width>
     <height>94</height>
    </rect>
   </property>
   <property name="title">
    <string>Viewing files</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="oneTabBtn">
      <property name="text">
       <string>View files in one tab</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="multiTabBtn">
      <property name="text">
       <string>View files in multiple tabs</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>671</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>File deletion</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="textLabel1">
      <property name="text">
       <string>When closing a view or viewing another file</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="deleteBtn">
      <property name="text">
       <string>Delete file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="keepBtn">
      <property name="text">
       <string>Don't delete file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="askBtn">
      <property name="text">
       <string>Ask</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="statusBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>230</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable status bar on initial load</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="activateBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>309</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Activate view tab when a new file is loaded</string>
   </property>
   <property name="shortcut">
    <string>Alt+N</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
